frontend/src/components/auth/Login.tsx

```diff
@@ -101,22 +129,47 @@ const Login = (): React.ReactElement => {
           <Box>
             <Text variant="display-sm-sb">Sign in to access courses</Text>
             <FormControl>
-              <Text variant="caption-md" marginTop="4vh" marginBottom="1vh">
-                Email Address
-              </Text>
+              <FormLabel
+                htmlFor="email"
+                variant="caption-md"
+                marginTop="4vh"
+                marginBottom="1vh"
+              >
+                Email address
+              </FormLabel>
               <Input
+                ref={emailRef}
                 type="email"
                 value={email}
                 placeholder="you@rowanhouse.ca"
-                onChange={(event) => setEmail(event.target.value)}
+                onChange={(event: ChangeEvent<HTMLInputElement>) =>
+                  setEmail(event.target.value)
```

Comment:
If the user modifies their email here, we should reset the contents of the password field. This is so the browser doesn't think the user submitted an autofilled email.